### PR TITLE
Add mpv icon

### PR DESCRIPTION
**Pull request by ChevySSinSD**
mpv

**Similarweb rank:** [GitHub 18.2k stars](https://github.com/mpv-player/mpv/)
  

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description

- Hex value from top bar of http://mpv.io
- Generated svg from unofficial logo in Wikipedia commons and adjusted proportion of triangle to match the official logo in Wikipedia comments (png file)

### CODE REVIEW

**Comment by adamrusted** [comment]
Official PSD files are available at https://github.com/mpv-player/mpv/commit/303096bcb2b5f52ea51d5d7ddb738689b08a0a2f (the dropbox link in the commit description). Can someone with access to Photoshop convert that into a decent version for us?

File: _data/simple-icons.json [hunk 1]
```diff
@@ -6588,6 +6588,11 @@
             "source": "https://mozilla.design/mozilla/",
             "guidelines": "https://mozilla.design/mozilla/"
         },
+        {
+            "title": "mpv",
+            "hex": "520053",
+            "source": "https://commons.wikimedia.org/wiki/File:Mpv_logo_(official).png"
```
Comment (PratyushJayachandran) [inline _data/simple-icons.json:6594]:
```suggestion
            "source": "https://github.com/mpv-player/mpv/blob/da400ed3a9b54408d1b5112855a7281380a1ef52/etc/mpv.svg"
```
SVG from GitHub repo

File: _data/simple-icons.json [hunk 2]
```diff
@@ -6588,6 +6588,11 @@
             "source": "https://mozilla.design/mozilla/",
             "guidelines": "https://mozilla.design/mozilla/"
         },
+        {
+            "title": "mpv",
+            "hex": "520053",
```
Comment (PratyushJayachandran) [inline _data/simple-icons.json:6593]:
```suggestion
            "hex": "691F69",
```
They seem to be using different shades of purple on their website. I took the one used by the play button on the [SVG found here](https://github.com/mpv-player/mpv/blob/da400ed3a9b54408d1b5112855a7281380a1ef52/etc/mpv.svg)

File: icons/mpv.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>mpv</title><path d="M10.161 9.29v5.624l4.838-2.88zm2.952-6.816a8.457 8.47 0 0 0-8.413 8.47 8.457 8.47 0 0 0 8.456 8.47 8.457 8.47 0 0 0 8.457-8.47 8.457 8.47 0 0 0-8.457-8.47 8.457 8.47 0 0 0-.044 0zM12.041 6.64a5.433 5.442 0 0 1 .02 0 5.433 5.442 0 0 1 5.433 5.442 5.433 5.442 0 0 1-5.434 5.441 5.433 5.442 0 0 1-5.432-5.441 5.433 5.442 0 0 1 5.414-5.442zM12.002 0A12 12 0 0 0 0 12a12 12 0 0 0 12 12 12 12 0 0 0 12-12A12 12 0 0 0 12 0zm.262.491a11.142 11.142 0 0 1 .049 0 11.142 11.142 0 0 1 11.143 11.142 11.142 11.142 0 0 1-11.142 11.142A11.142 11.142 0 0 1 1.17 11.633 11.142 11.142 0 0 1 12.263.491z"/></svg>
```
Comment (PratyushJayachandran) [inline icons/mpv.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>mpv</title><path d="M12 0C5.373 0 0 5.373 0 12s5.373 12 12 12 12-5.373 12-12S18.627 0 12 0zm.312 22.775c-6.153 0-11.142-4.988-11.142-11.142S6.16.491 12.312.491c6.154 0 11.142 4.989 11.142 11.142s-4.988 11.142-11.142 11.142zm.643-20.464a8.587 8.587 0 1 0 0 17.174 8.587 8.587 0 0 0 0-17.174zm-1.113 15.257a5.517 5.517 0 1 1 0-11.034 5.517 5.517 0 0 1 0 11.034zm-1.399-7.995L14.4 11.97l-3.957 2.518V9.573z"/></svg>
```

Comment (PratyushJayachandran) [reply icons/mpv.svg:1]: Remade from source available here:
https://github.com/mpv-player/mpv/blob/da400ed3a9b54408d1b5112855a7281380a1ef52/etc/mpv.svg

**Review by PratyushJayachandran — CHANGES_REQUESTED** [review]

**Comment by PratyushJayachandran** [comment]
> Official PSD files are available at [mpv-player/mpv@303096b](https://github.com/mpv-player/mpv/commit/303096bcb2b5f52ea51d5d7ddb738689b08a0a2f) (the dropbox link in the commit description). Can someone with access to Photoshop convert that into a decent version for us?

I found the official source https://github.com/mpv-player/mpv/blob/da400ed3a9b54408d1b5112855a7281380a1ef52/etc/mpv.svg
Please review

File: icons/mpv.svg [hunk 4]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>mpv</title><path d="M12 0C5.373 0 0 5.373 0 12s5.373 12 12 12 12-5.373 12-12S18.627 0 12 0zm.312 22.775c-6.153 0-11.142-4.988-11.142-11.142S6.16.491 12.312.491c6.154 0 11.142 4.989 11.142 11.142s-4.988 11.142-11.142 11.142zm.643-20.464a8.587 8.587 0 1 0 0 17.174 8.587 8.587 0 0 0 0-17.174zm-1.113 15.257a5.517 5.517 0 1 1 0-11.034 5.517 5.517 0 0 1 0 11.034zm-1.399-7.995L14.4 11.97l-3.957 2.518V9.573z"/></svg>
```
Comment (PratyushJayachandran) [inline icons/mpv.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>mpv</title><path d="M12 0C5.373 0 0 5.373 0 12s5.373 12 12 12 12-5.373 12-12S18.627 0 12 0zm.312 22.775c-6.153 0-11.142-4.988-11.142-11.142S6.16.491 12.312.491c6.154 0 11.142 4.989 11.142 11.142s-4.988 11.142-11.142 11.142zm.643-20.464a8.587 8.587 0 1 0 0 17.174 8.587 8.587 0 0 0 0-17.174zm-1.113 15.257a5.517 5.517 0 1 1 0-11.034 5.517 5.517 0 0 1 0 11.034zm-1.399-7.995L14.4 11.97l-3.957 2.518V9.573z"/></svg>
```

Comment (PratyushJayachandran) [inline icons/mpv.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>mpv</title><<path d="m10.4 9.6 4 2.4-4 2.5V9.6zM24 12c0 6.6-5.4 12-12 12S0 18.6 0 12 5.4 0 12 0s12 5.4 12 12zm-.5-.4c0-6.2-5-11.1-11.1-11.1-6.2 0-11.1 5-11.1 11.1s5 11.1 11.1 11.1c6.1.1 11.1-4.9 11.1-11.1zm-2-.7c0 4.7-3.8 8.6-8.6 8.6-4.7 0-8.6-3.8-8.6-8.6S8.2 2.3 13 2.3c4.7 0 8.5 3.9 8.5 8.6zm-4.1 1.2c0-3-2.5-5.5-5.5-5.5s-5.5 2.5-5.5 5.5 2.5 5.5 5.5 5.5 5.5-2.5 5.5-5.5z"/></svg>
```
Corrected icon fetched from official source.

**Review by PratyushJayachandran — APPROVED** [review]

**Comment by PratyushJayachandran** [comment]
@adamrusted Please review this PR. I fixed the icon, fetched from official source and changed the color

File: icons/mpv.svg [hunk 5]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>mpv</title><<path d="m10.4 9.6 4 2.4-4 2.5V9.6zM24 12c0 6.6-5.4 12-12 12S0 18.6 0 12 5.4 0 12 0s12 5.4 12 12zm-.5-.4c0-6.2-5-11.1-11.1-11.1-6.2 0-11.1 5-11.1 11.1s5 11.1 11.1 11.1c6.1.1 11.1-4.9 11.1-11.1zm-2-.7c0 4.7-3.8 8.6-8.6 8.6-4.7 0-8.6-3.8-8.6-8.6S8.2 2.3 13 2.3c4.7 0 8.5 3.9 8.5 8.6zm-4.1 1.2c0-3-2.5-5.5-5.5-5.5s-5.5 2.5-5.5 5.5 2.5 5.5 5.5 5.5 5.5-2.5 5.5-5.5z"/></svg>
```
Comment (PratyushJayachandran) [inline icons/mpv.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>mpv</title><path d="m10.4 9.6 4 2.4-4 2.5V9.6zM24 12c0 6.6-5.4 12-12 12S0 18.6 0 12 5.4 0 12 0s12 5.4 12 12zm-.5-.4c0-6.2-5-11.1-11.1-11.1-6.2 0-11.1 5-11.1 11.1s5 11.1 11.1 11.1c6.1.1 11.1-4.9 11.1-11.1zm-2-.7c0 4.7-3.8 8.6-8.6 8.6-4.7 0-8.6-3.8-8.6-8.6S8.2 2.3 13 2.3c4.7 0 8.5 3.9 8.5 8.6zm-4.1 1.2c0-3-2.5-5.5-5.5-5.5s-5.5 2.5-5.5 5.5 2.5 5.5 5.5 5.5 5.5-2.5 5.5-5.5z"/></svg>
```

**Review by adamrusted — CHANGES_REQUESTED** [review]
I'm seeing [some difference](https://github.com/simple-icons/simple-icons/assets/1830867/aa641f63-9728-4f59-b423-1b759be50bb7) when comparing the source in white to the PR in black. Could you take another look please @PratyushJayachandran / @ChevySSinSD?

File: icons/mpv.svg [hunk 6]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>mpv</title><path d="m10.4 9.6 4 2.4-4 2.5V9.6zM24 12c0 6.6-5.4 12-12 12S0 18.6 0 12 5.4 0 12 0s12 5.4 12 12zm-.5-.4c0-6.2-5-11.1-11.1-11.1-6.2 0-11.1 5-11.1 11.1s5 11.1 11.1 11.1c6.1.1 11.1-4.9 11.1-11.1zm-2-.7c0 4.7-3.8 8.6-8.6 8.6-4.7 0-8.6-3.8-8.6-8.6S8.2 2.3 13 2.3c4.7 0 8.5 3.9 8.5 8.6zm-4.1 1.2c0-3-2.5-5.5-5.5-5.5s-5.5 2.5-5.5 5.5 2.5 5.5 5.5 5.5 5.5-2.5 5.5-5.5z"/></svg>
```
Comment (PratyushJayachandran) [inline icons/mpv.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>mpv</title><path d="M12 0C5.373 0 0 5.373 0 12s5.373 12 12 12 12-5.373 12-12S18.627 0 12 0zm.312 22.775c-6.153 0-11.142-4.988-11.142-11.142S6.16.491 12.312.491c6.154 0 11.142 4.989 11.142 11.142s-4.988 11.142-11.142 11.142zm.643-20.464a8.587 8.587 0 1 0 0 17.174 8.587 8.587 0 0 0 0-17.174zm-1.113 15.257a5.517 5.517 0 1 1 0-11.034 5.517 5.517 0 0 1 0 11.034zm-1.399-7.995L14.4 11.97l-3.957 2.518V9.573z"/></svg>
```
This should fix the issues

**Comment by PratyushJayachandran** [comment]
> I'm seeing [some difference](https://github.com/simple-icons/simple-icons/assets/1830867/aa641f63-9728-4f59-b423-1b759be50bb7) when comparing the source in white to the PR in black. Could you take another look please @PratyushJayachandran / @ChevySSinSD?

Please review again :)

**Review by adamrusted — APPROVED** [review]

**Pull request merged**